gradle/libs.versions.toml

```
@@ -40,8 +40,8 @@ pluginPackages-aspectj-postCompileWeaving = "io.freefair.gradle:aspectj-plugin:6
 pluginPackages-checkerFramework = "org.checkerframework:checkerframework-gradle-plugin:0.6.19"
 pluginPackages-grgit = "org.ajoberstar.grgit:grgit-gradle:5.0.0"
 pluginPackages-jetbrains-changelog = "org.jetbrains.intellij.plugins:gradle-changelog-plugin:2.0.0"
-pluginPackages-jetbrains-grammarkit = "org.jetbrains.intellij.plugins:gradle-grammarkit-plugin:2021.2.2"
-pluginPackages-jetbrains-intellij = "org.jetbrains.intellij.plugins:gradle-intellij-plugin:1.9.0"
+pluginPackages-jetbrains-grammarkit = "org.jetbrains.intellij.plugins:gradle-grammarkit-plugin:2022.3"
```

Let's revert this one for the moment, just to merge what's working right now....
```suggestion
pluginPackages-jetbrains-grammarkit = "org.jetbrains.intellij.plugins:gradle-grammarkit-plugin:2021.2.2"
```